modules/ROOT/pages/monitor-api-conformance.adoc

```diff
@@ -1,99 +1,164 @@
 = Monitoring API Conformance
 
-After you configure governance profiles for your APIs and the resulting governance data is available, you can monitor and report on your organization's API specifications' conformance success and failure.  
+After you configure governance profiles for your APIs and the resulting governance data is available, you can monitor and report on your organization's API conformance.  
 
-== Status Information on the Console
+== Test Profile Settings Using Draft Profiles
 
-You can view the latest status information in the console.
+Consider using draft profiles to test profile settings. This enables you to limit the scope of the profiles to just what you need before alerting the API owners about identified API conformance issues. 
 
-// Screenshot of the console with callouts
-
-include::_partials/api-gov-console-scap-callouts.adoc[]
-
-NOTE: Only API Governance and organization administrators can create, edit, and delete profiles.
-
-// Definition of Governance Profiles
-
-include::general::partial$api-gov-concepts.adoc[tag=governance-profiles,leveloffset=+2]
-
-// Definition of Governance Profile Status
-
-include::general::partial$api-gov-concepts.adoc[tag=governance-profile-status,leveloffset=+2]
+// Screenshot of a draft profile's page
 
-// Definition of Governed APIs
+image::api-gov-draft-profile-view.png[Screenshot of the API conformance information for a selected draft profile]
```

Comment:
I think the [guidance](https://confluence.internal.salesforce.com/display/DOCTEAM/Accessibility+Hub#AccessibilityHub-TipsforWritingEffectiveAltText) is to not include words like screenshot or image in the description.

Reply:
Yes, but it says "if any such words are necessary for context or clarity, it’s OK to include them". I understand not using the word image since the reader says image, but it might not be clear what the image is when it's a screenshot. I'll defer to your guidance, but typically like the option to choose whether I take the exception.